db/migrate/20210222214631_add_cavc_mdr_federal_circuit.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddCavcMdrFederalCircuit < Caseflow::Migration
+  def change
+    add_column :cavc_remands, :federal_circuit, :boolean,
+               comment: "Whether the case has been appealed to the US Court of Appeal for the Federal Circuit"
```

Comment:
```suggestion
               comment: "Whether the case has been appealed to the US Court of Appeals for the Federal Circuit"
```

Trivial nit, but they apparently pluralize "Appeals" in their name: http://www.cafc.uscourts.gov/